app/models/group.rb

```diff
@@ -47,6 +46,16 @@ def add_leader_to_members
     members << leader unless members.include? leader
   end
 
+  def validate_suite_size_inclusion
+    if draw.present?
+      return if draw.suite_sizes.include? size
+      errors.add :size, 'must be a suite size included in the draw'
+    else
+      return if SuiteSizesQuery.call.include? size
+      errors.add :size, 'must be a valid suite size'
+    end
+  end
```

Comment:
if we end up with more draw presence conditionals, we should consider creating a `DrawlessGroup` object to contain all of this behavior, but this is fine for now